packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1272,10 +1272,8 @@ describe('shallow', () => {
       expect(wrapper.find('.foo')).to.have.lengthOf(1);
 
       wrapper[sym('__renderer__')].batchedUpdates(() => {
-        wrapper.setProps({ id: 'bar', foo: 'bla' }, () => {
-          expect(wrapper.find('.bar')).to.have.lengthOf(1);
-        });
```

Comment:
I think callback is not supported for ShallowRendering
https://github.com/airbnb/enzyme/blob/master/packages/enzyme/src/ShallowWrapper.js#L381

Reply:
no, but it probably should be.

Reply:
does it mean I should change `setProps` function in `ShallowWrapper`?
Because right now `setProps()` only has on parameter for the props to update, right?